plugins/filter/lib/lib.pro

```diff
@@ -0,0 +1,40 @@
+TEMPLATE = lib
+TARGET = voicecall-filter-plugin
+
+DEFINES += FILTER_SHARED
+CONFIG += hide_symbols
+
+CONFIG += shared \
+          no_install_prl \
+          link_pkgconfig \
+          c++11
```

Comment:
C++11 already the default. We have a bunch of unnecessary instances of these here and there.

Reply:
Ok, nice, removed.